docs/json/mperday.json

```diff
@@ -0,0 +1,68 @@
+[
```

Comment:
I'm not too familiar with these. Could you briefly explain these set of json files?

Reply:
I've removed them in the latest commit. Was used to initialize data

Reply:
Sure thing